docs/attributes-registry/db.md

```diff
@@ -0,0 +1,152 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Database
+
+## Database Attributes
+
+<!-- semconv registry.db(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `db.cassandra.consistency_level` | string | The consistency level of the query. Based on consistency values from [CQL](https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/dml/dmlConfigConsistency.html). | `all` |
+| `db.cassandra.coordinator.dc` | string | The data center of the coordinating node for a query. | `us-west-2` |
+| `db.cassandra.coordinator.id` | string | The ID of the coordinating node for a query. | `be13faa2-8574-4d71-926d-27f16cf8a7af` |
+| `db.cassandra.idempotence` | boolean | Whether or not the query is idempotent. |  |
+| `db.cassandra.page_size` | int | The fetch size used for paging, i.e. how many rows will be returned at once. | `5000` |
```

Comment:
Do you think it would be better to read if these would also split "by db"? Now they are all together and if I'm looking for a single DB (say MongoDb) it's a bit hard to read/find things.

Reply:
@joaopgrassi Grouped the attributes by technology. Hope that looks better.